vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -89,6 +89,9 @@ using PendingContributionsTotalCallback = std::function<void(double)>;
 using PendingContributionInfoListCallback =
     std::function<void(type::PendingContributionInfoList)>;
 
+using UnverifiedPublishersCallback =
+    std::function<void(std::vector<std::string>&&)>;
```

Comment:
Curious: why the rvalue reference here (as opposed to a reference or a value)?

Reply:
People often forget to `std::move<>()` their stuff, IMO.
Even if they don't, there's a good chance they end up with two moves instead of one, even when using rvalues (not taking into consideration copy elision/(N)RVO here), e.g.:
```
struct S {
    void f(std::vector<std::string> v2) { // pass by value
        v1 = std::move(v2);
    }
	
private:
    std::vector<std::string> v1;
};

S{}.f(std::vector<std::string>{}); // move constructing v2, then move assigning v2 into v1
```

Whereas it could have been only one move:
```
struct S {
    void f(std::vector<std::string>&& v2) { // pass by rvalue reference
        v1 = std::move(v2);
    }
	
private:
    std::vector<std::string> v1;
};

S{}.f(std::vector<std::string>{}); // move assigning v2 into v1
```
On the other hand, rvalue references bind to rvalues only, so you cannot accidentally pass in an lvalue and end up copying it:
```
std::vector<std::string> lvalue{};
S{}.f(lvalue); // cannot bind rvalue reference to lvalue (literally :D)
```
I could have used a reference-to-`const` as well, but then I would have had to make a copy somewhere, which I then could move-capture into the lambda in `Unverified::FetchInfoForUnverifiedPublishers()`.